src/suites/cts/command_buffer/copies.spec.ts

```diff
@@ -109,3 +109,67 @@ g.test('b2t2t2b', async t => {
 
   t.expectContents(dst, data);
 });
+
+g.test('imageBitmapToTexture', async t => {
```

Comment:
This is not a command_buffer test anymore because copyImageBitmapToTexture was moved to the queue.
Could you move these new tests into a new file, say, `src/suites/cts/copyImageBitmapToTexture.spec.ts` for now.

Reply:
Sure. I classify it into the copy buffer. But you're right, under commadnBuffer test suite is not proper.Will do it.

Reply:
Done.